pages/[...slug].tsx

```diff
@@ -351,11 +351,11 @@ export const GenericPage: React.FC<GenericPageProps> = ({
     </PageBanner>
   )
 
-  const isTabletOrMobile = useMediaQuery({
-    query: `(max-width: ${breakpoint('m').breakpoint})`,
+  const isDesktop = useMediaQuery({
+    query: `(min-width: ${breakpoint('m').breakpoint})`,
   })
 
-  const navigation = isTabletOrMobile ? mobileNavigation : desktopNavigation
+  const navigation = isDesktop ? desktopNavigation : mobileNavigation
```

Comment:
Not one for now - but there is a fair bit of duplication in this file and the homepage one that could probably do with being abstracted out. I'll add a note to board about that.